x/evm/keeper/msg_server.go

```diff
@@ -750,6 +750,12 @@ func (s msgServer) VoteConfirmDeposit(c context.Context, req *types.VoteConfirmD
 		sdk.NewAttribute(sdk.AttributeKeyModule, types.ModuleName),
 		sdk.NewAttribute(sdk.AttributeKeyAction, types.AttributeValueVote),
 		sdk.NewAttribute(types.AttributeKeyValue, strconv.FormatBool(voteValue.Value)),
+		sdk.NewAttribute(types.AttributeKeyChain, chain.Name),
```

Comment:
Do we have any consumers asking for these?

Reply:
Yep, explorer has had issues due to inconsistent info in these specific events.